### DISCUSSION TITLE
Video that shouldn't be inverted. Cannot fix with dev tools?

### DISCUSSION THREAD

**Discussion by mrmoto** [darkreader/darkreader#9721] [Q&A]
Hello, 

www.coursera.org/.../some-video-page has a video playing with colors inverted that shouldn't be. 

element Inspection:
<video class="vjs-tech" crossorigin="anonymous" preload="auto" autoplay="" tabindex="-1" aria-label="Video: Black Box vs. White Box" id="vjs_video_3_html5_api"....>

Seems like the class descriptor would be best to address for other video pages...

Dev tools JS code added:
{
            "url": "www.coursera.org",
            "noinvert": [
                ".vjs-tech"
            ]
},

Nothing. Still inverts. 
<img width="282" alt="Screen Shot 2022-09-01 at 12 58 29 PM" src="https://user-images.githubusercontent.com/8846924/188004081-8f1c8183-7d40-4121-8e47-3a982ba16020.png">
<img width="1007" alt="Screen Shot 2022-09-01 at 12 56 06 PM" src="https://user-images.githubusercontent.com/8846924/188004088-226ba6db-0e0a-4d89-9f28-2be317559912.png">
Pretty new to programming, but I cannot get this to work. Seems simple, I'm sure I'm just missing the simple error I'm making.

**Comment by Gusted** [comment]
The inversion fixes don't use JSON. You should be using https://github.com/darkreader/darkreader/blob/main/src/config/inversion-fixes.config

**Comment by mrmoto** [comment]
Thanks for the reply. I'll add it to the GH page as well, sans JSON of
course :-)

I was attempting to use noinvert on the DevTools widget in my desktop
chrome application, which seems to be JSON.

Sorry for the confusion.

Scott Roberts
(925) 818-5388 m
(510) 288-8414 o
***@***.***
